src/elements/content-sidebar/__tests__/SidebarPanels.test.js

```diff
@@ -56,6 +74,27 @@ describe('elements/content-sidebar/SidebarPanels', () => {
             expect(wrapper.exists(sidebar)).toBe(true);
         });
 
+        test.each`
```

Comment:
Should we test the case that the path selected panel is different than the saved panel? Should the path selected panel take precedence?

Reply:
Right, I'll add that test.